tests/test_utils.py

```diff
@@ -0,0 +1,88 @@
+import json
+import os
+import unittest
+
+from mock import Mock, mock
+from mockredis import mock_redis_client
+from redis import ConnectionError, RedisError
+
+from app import utils, server
+
+
+class TestInventoryServer(unittest.TestCase):
+  def setUp(self):
+    utils.Redis = Mock(return_value=mock_redis_client)
+    mock_redis_client.ping = Mock(return_value=True)
+    utils.connect_to_redis = Mock(side_effect=utils.connect_to_redis)
+
+  def test_info_is_valid(self):
+    info = {server.TYPE: 'used', server.QUANTITY: 20}
```

Comment:
Could you define constant strings and use them instead? just like what's in the `test_server`